backend/dal/Migrations/v01.07.02/Up/PostUp/01-ViewProperties.sql

```diff
@@ -0,0 +1,213 @@
+PRINT 'Updating View_Properties - Updating Assessment'
+GO
+-- This script provides a way to union both parcels and buildings into a single result of properties.
+-- Updating view to support many-to-many parcels and buildings.TEMPORARY
+CREATE OR ALTER VIEW dbo.[View_Properties] AS
+SELECT
+    p.[Id]
+    , p.[RowVersion]
+    , [PropertyTypeId] =  (SELECT CASE WHEN sp.[ParcelId] IS NOT NULL THEN 2 ELSE 0 END)
+    , p.[ClassificationId]
+    , [Classification] = c.[Name]
+    , p.[AgencyId]
+    , [Agency] = ISNULL(pa.[Name], a.[Name])
+    , [AgencyCode] = ISNULL(pa.[Code], a.[Code])
+    , [SubAgency] = (
+        SELECT CASE WHEN pa.[Id] IS NOT NULL
+            THEN a.[Name]
+            ELSE NULL
+        END)
+    , [SubAgencyCode] = (
+        SELECT CASE WHEN pa.[Id] IS NOT NULL
+            THEN a.[Code]
+            ELSE NULL
+        END)
+    , p.[AddressId]
+    , [Address] = TRIM(ISNULL(adr.[Address1], '') + ' ' + ISNULL(adr.[Address2], ''))
+    , [AdministrativeArea] = adr.[AdministrativeArea]
+    , [Province] = ap.[Name]
+    , adr.[Postal]
+    , p.[ProjectNumbers]
+    , p.[Name]
+    , p.[Description]
+    , p.[Location]
+    , p.[Boundary]
+    , p.[IsSensitive]
+    , p.[IsVisibleToOtherAgencies]
+
+    -- Parcel Properties
+    , p.[PID]
+    , p.[PIN]
+    , [LandArea] = p.[LandArea]
+    , [LandLegalDescription] = p.[LandLegalDescription]
+    , [Zoning] = p.[Zoning]
+    , [ZoningPotential] = p.[ZoningPotential]
+
+    -- Building Properties
+    , [ParcelId] = p.[Id]
+    , [BuildingConstructionTypeId] = 0
+    , [BuildingConstructionType] = null
+    , [BuildingFloorCount] = 0
+    , [BuildingPredominateUseId] = 0
+    , [BuildingPredominateUse] = null
+    , [BuildingOccupantTypeId] = 0
+    , [BuildingOccupantType] = null
+    , [BuildingTenancy] = null
+    , [RentableArea] = 0
+    , [LeaseExpiry] = null
+    , [OccupantName] = null
+    , [TransferLeaseOnSale] = CAST(0 AS BIT)
+
+    , [AssessedLand] = eas.[Value]
+    , [AssessedLandDate] = eas.[Date]
+    , [AssessedBuilding] = eim.[Value]
+    , [AssessedBuildingDate] = eim.[Date]
+    , [Market] = ISNULL(fe.[Value], 0)
+    , [MarketFiscalYear] = fe.[FiscalYear]
+    , [NetBook] = ISNULL(fn.[Value], 0)
+    , [NetBookFiscalYear] = fn.[FiscalYear]
+FROM dbo.[Parcels] p
+LEFT JOIN dbo.[ParcelParcels] sp ON p.[Id] = sp.[ParcelId]
```

Comment:
I think this will actually have unintentional effect of creating multiple copies of the same parcel and will result in not correctly apply the correct `PropertyType` to the parent parcel.

Reply:
This should be LEFT JOIN dbo.[ParcelParcels] sp ON p.[Id] = sp.[SubdivisionId].

Parcels that have their id being the same as a subdivisionId are subdivisions.

I suppose the duplication would happen if there were multiple parcelParcel rows with the same subdivisionId. Could get around this by joining to a select distinct subdivisionId from parcelParcel instead of just joining to the parcelParcel table.

Reply:
I think you can avoid this by not using this relationship to decide on the `ProperytType`.  Just include the `PropertyType` to the table, or add a more complex relationship `ParcelType` and `BuildingType`

Reply:
I'll add in the property type, but I think I'll keep the two arrays. Only using one array works, but ends up being similarly confusing and isn't worth the time investment to rewrite everything in my opinion.